README.md

```diff
@@ -34,7 +34,7 @@ For running the client:
 Server:
 
 	$ sudo apt-get install python-dev
-    $ sudo pip install -r requirements_server.txt
+  $ sudo pip install -r requirements_server.txt
```

Comment:
There is a small problem with this, it seems that it does not render properly, similar to the below $ cmake --version. But no problem, we'll fix it later, thanks for the contribution!